src/components/SubNavigationMenu/SubNavigatioMenu.stories.tsx

```diff
@@ -0,0 +1,97 @@
+/** @jsxImportSource theme-ui */
+import SubNavigationMenu, { SubNavigationMenuProps } from "./SubNavigationMenu"
+import { Meta, Story } from "@storybook/react/types-6-0"
+import { useState } from "react"
+
+export default {
+  title: "Design System/SubNavigationMenu",
+  component: SubNavigationMenu,
+  decorators: [
+    (Story) => (
+      <div sx={{ maxWidth: "auto", width: "fit-content" }}>
+        <Story />
+      </div>
+    ),
+  ],
+} as Meta
+
+const nonCollapsed = ["License types", "Licenses"]
+const collapsed = ["LT", "L"]
+
+const Template: Story<SubNavigationMenuProps> = (props) => {
+  const [selected, setSelected] = useState("License types")
+  const [items, setItems] = useState(nonCollapsed)
+  return (
+    <SubNavigationMenu
+      {...props}
+      onCollapseButtonClick={(isCollapsed) => {
+        setItems(isCollapsed ? collapsed : nonCollapsed)
+      }}
+    >
+      {items.map((item, index) => (
+        <SubNavigationMenu.MenuItem
+          key={item}
+          label={item}
+          onClick={(event) => {
+            setSelected(event.currentTarget.textContent as string)
+          }}
+          selected={[collapsed[index], nonCollapsed[index]].includes(selected)}
+        />
+      ))}
+    </SubNavigationMenu>
+  )
+}
+
+const TemplateWithChildren: Story<SubNavigationMenuProps> = (props) => {
+  const [selected, setSelected] = useState("License types")
+  const [items, setItems] = useState(nonCollapsed)
+
+  return (
+    <SubNavigationMenu
+      {...props}
+      onCollapseButtonClick={(isCollapsed) => {
+        setItems(isCollapsed ? collapsed : nonCollapsed)
+      }}
+    >
+      {items.map((item, index) => (
+        <SubNavigationMenu.MenuItem
+          key={item}
+          label={item}
+          onClick={(event) => {
+            setSelected(event.currentTarget.textContent as string)
+          }}
+          selected={[collapsed[index], nonCollapsed[index]].includes(selected)}
+        >
+          <a
+            sx={{
+              py: 2,
+              paddingLeft: 3,
+              paddingRight: 9,
+              whiteSpace: "nowrap",
+              width: "100%",
+            }}
```

Comment:
Isn't it possible for these styles be the default when using `children`? So that the dev can just use a `<Link>` component, or a string even, and it looks as expected without further styling needed?